core/lib/spree/core/number_generator.rb

```diff
@@ -1,18 +1,13 @@
 module Spree
   module Core
     class NumberGenerator < Module
-      BASE           = 10
       DEFAULT_LENGTH = 9
-      NUMBERS        = (0..9).to_a.freeze
-      LETTERS        = ('A'..'Z').to_a.freeze
-
-      attr_accessor :prefix, :length
 
       def initialize(options)
         @random     = Random.new
```

Comment:
It looks like you're no longer using @random in this class with the secure random approach.